Graphs/BarGraph/index.js

```diff
@@ -117,6 +119,9 @@ const BarGraph = (props) => {
         }
     );
 
+    if (!parsedData || !parsedData.length) {
+        return renderMessage({ message: "No data to visualize", id, classes });
```

Comment:
Is this change required for this bug?
In any case, can you use `NO_DATA_FOUND` from constants?

Reply:
Done